README.md

```diff
@@ -1,8 +1,31 @@
 ## Passage Licensing Runtime  
 
-Passage Licensing Runtime layer includes interfaces and its implementation for Eclipse Equinox
+The Passage Licensing Runtime is a part of the Eclipse Passage licensing system. 
+This component verifies that the software license is valid in accordance with the specified license requirements.
+The Equinox-based implementation includes various validators for the subscription-based, node locked and other types of licenses.
 
+### How to use
 
-  Copyright (c) 2018 ArSysOp
+Just include the Passage Licensing Runtime bundles in your product.
 
+### How it works
 
+The Passage Licensing Runtime works inside your product installed on the user side. Its basic steps are as follows:
+1. Get the system configuration specified in the Licensing Operator client and extract the identifiers of features to be controlled.
+2. Get the license requirements. By default, the tool looks for the license files located in the user's home subdirectory.
+3. Compare the license specified configuration with the current system state (the current date, node id, etc) to detect invalid cases.
+4. Pass the system configuration requirements and the detected invalid cases to the decision maker.
+5. Call the required constraint function to show the license expiration message, limit certain functionality or disable everything.
```

Comment:
We need to keep original lifecycle as is, I tried to formulate it again with some comments

1) resolve ConfigurationRequirement(s): what is required by your configuration?
2) mine LicensingCondition(s): what is granted by your license?
3) evaluate LicensingCondition(s) to receive FeaturePermission(s): is granted still valid?
4) examine requirements and permissions to compose RestrictionVerdict(s): required - granted=restricted
5) execute RestrictionVerdict(s) to realize restrictions: if restricted is empty all is ok